### PR TITLE
[Ready!] The Slimening - Reworking slimepeople to be less like humans without unbalanced advantages.

### PR DESCRIPTION
**Pull request by jakeramsay007**
:cl: RealDonaldTrump
add: Slimepeople (And all types, including Jelly, Xenobio Slimeperson, Stargazer and Luminescent) now have the ability to utilise the 'Alter Form' ability, allowing them to change most things about their body's form.
add: Re-added the slime split and bodyswap ability to Xenobiological Slimepeople, only obtainable through xenobio.
add: Slimepeople can now select tails, taur bodies and ears at roundstart. YOU WILL NEED TO SET YOUR COLOURS; otherwise you'll be rainbow coloured and not exactly slime-like. And nobody wants that.
config: Changed the ID for the roundstart slimepeople (Without the body swap and slime split abilities) to slimeperson. This will need set in the config on Jay's end.
/:cl:

As stated above, I've given them the ability to change form. This is at a cost, however; slimepeople no longer have protection from burns (previously they took 25% of the damage from being burnt or otherwise heat damaged, such as with laser guns). They also take 3x more damage from the cold, so if you're in space as a slimeperson and unprotected? Expect to die very, very quickly. Xenobiological Slimepeople have their burn protection re-added because they're supposed to be a straight up upgrade to regular slimepeople.

Xenobiological Slimepeople also now have their virus immunity back, however this is subject to change and balancing later on.

Thanks goes to @fludd12 for her previous pull request, as it helped me out immensely with figuring out what needed to be changed, as well as her input during the minor rework.

### CODE REVIEW

**Review by deathride58 — CHANGES_REQUESTED** [review]
Toss the new species and the ability into furrypeople.dm, otherwise roundstart slimes are gonna be gone every single time jellypeople.dm gets synced

File: code/modules/mob/living/carbon/human/species_types/jellypeople.dm
Comment (deathride58) [inline code/modules/mob/living/carbon/human/species_types/jellypeople.dm]: Send this to upstream, this is a fix for an upstream issue.

File: code/modules/mob/living/carbon/human/species_types/jellypeople.dm
Comment (deathride58) [inline code/modules/mob/living/carbon/human/species_types/jellypeople.dm]: add //CIT CHANGE

File: code/modules/mob/living/carbon/human/species_types/jellypeople.dm
Comment (deathride58) [inline code/modules/mob/living/carbon/human/species_types/jellypeople.dm]: add //CIT CHANGE

File: code/modules/mob/living/carbon/human/species_types/jellypeople.dm
Comment (deathride58) [inline code/modules/mob/living/carbon/human/species_types/jellypeople.dm]: add //CIT CHANGE

File: code/modules/mob/living/carbon/human/species_types/jellypeople.dm
Comment (deathride58) [inline code/modules/mob/living/carbon/human/species_types/jellypeople.dm]: add //CIT CHANGE

File: code/modules/mob/living/carbon/human/species_types/jellypeople.dm
Comment (deathride58) [inline code/modules/mob/living/carbon/human/species_types/jellypeople.dm]: You can modularize the species name change

File: code/modules/mob/living/carbon/human/species_types/jellypeople.dm
Comment (deathride58) [inline code/modules/mob/living/carbon/human/species_types/jellypeople.dm]: add //CIT CHANGE

File: code/modules/mob/living/carbon/human/species_types/jellypeople.dm
Comment (deathride58) [inline code/modules/mob/living/carbon/human/species_types/jellypeople.dm]: add //CIT CHANGE - adds HAIR and FACEHAIR to species traits

**Review by deathride58 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/species_types/jellypeople.dm
Comment (jakeramsay007) [reply code/modules/mob/living/carbon/human/species_types/jellypeople.dm]: Are you wanting //CIT CHANGE added to the end of it?

**Review by deathride58 — DISMISSED** [review]
Just needs config updates from Jay

**Comment by jakeramsay007** [comment]
Actually, gonna wait till I get a UI sprite to replace the godawful look of the current one

**Comment by geemiesif** [comment]
Viro immune wont last long tbh , refactor cold damage to 125% more vs 200% more , and dont add bonus to the xenobio slimepeople to avoid people crying for nerfs .

**Comment by jakeramsay007** [comment]
@geemiesif The massive cold damage they take is the downside to being a slimeperson. This is actually what was intended when they were added, but the heat/burn resistance cancelled it out unintentionally.

**Comment by jakeramsay007** [comment]
Do not merge until tgstation/tgstation/pull/35725 is merged

**Comment by fludd12** [comment]
Could you re-enable slime splitting? Also, how's this as an icon?
image
It's the base slime `grow` icon used when you're a literal slime, but with kitty ears.

I can send it through discord, if you like.

**Review by deathride58 — APPROVED** [review]
Needs config update from Jay. Code's good.

**Comment by yourdoom9898** [comment]
@fludd12 Slime splitting would be incredibly broken for a round-start species, especially after TG buffed it so that you can body swap after being knocked into crit. All you would need to do is get a bunch of sugar and chug it, then find somewhere safe in maint, and then you effectively have free extra lives.

**Comment by jakeramsay007** [comment]
@fludd12 Slime splitting is re-enabled for xenobio slimepeople, not for roundstart slimepeople.

**Comment by fludd12** [comment]
Yeh, that's what I meant.

**Pull request merged**
